cura/Settings/ProfilesModel.py

```diff
@@ -155,14 +164,24 @@ def _recomputeItems(self):
 
         # Now all the containers are set
         for item in containers:
-            profile = container_registry.findContainers(id=item["id"])
+            profile = container_registry.findContainers(id = item["id"])
+
+            # when the profile is not supported

```

Comment:
I suggest to better explain in the comments the difference between this case and the one that was introduced below.

Reply:
The difference is a not existing profile (in the container registry) vs an existing profile that has the not supported flag (meta data field). Personally I think the `if not profile:` makes it quite clear. It handles cases where the profile does not exist in the container registry, something that is (or should?) be done for every case where we're fetching containers from the registry.